docs/using-the-snowplow-console/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Using Snowplow Console"
+title: "Administration"
```

Comment:
I think this makes it quite unclear what this section relates to now. Maybe chose a different name, or add the enterprise tag at this level?

Reply:
To be fair, the previous title was not accurate either, because it only covered very specific aspects of using the Console (...administration). How about this?
<img width="309" alt="image" src="https://github.com/snowplow/documentation/assets/2670454/204478be-b8a3-4f53-8c4d-42a3b3ff9f7b">